code/game/objects/items/devices/autopsy.dm

```diff
@@ -185,23 +164,18 @@
 
 	if(target_name != M.name)
```

Comment:
This should probably be `realname` so doofy stuff doesn't happen if you put gas masks on the cadavers but eh that's a matter for another day